setup.py

```diff
@@ -74,7 +74,7 @@
     'invenio-oaiharvester==1.0.0a2',
     'invenio-utils==0.2.0',  # Not fully Invenio 3 ready
     'invenio>=3.0.0a1,<3.1.0',
-    'inspire-crawler~=0.2.7',
+    'inspire-crawler~=0.0,>=0.2.7',
```

Comment:
Yeah, I agree with @david-caro: let's uniformize (cit.) to the pattern that we're already using.